app/battle/fight.py

```diff
@@ -0,0 +1,17 @@
+from app.preparation.create_knight import Knight
+
+
+def battle(battlers: list[Knight]) -> None:
+
+    # 1 Lancelot vs Mordred:
+    battlers[0].hp -= battlers[2].power - battlers[0].protection

```

Comment:
battlers is a dict but not list
![image](https://user-images.githubusercontent.com/86322722/234833180-62ef1b2f-a438-4053-8c18-a7de86c29a2a.png)